docs/odh/oci-model-storage.md

```diff
@@ -88,40 +294,19 @@ tree
 Create the OCI container image with Podman, and upload it to a registry. For
 example, using Quay as the registry:
 ```shell
-podman build --format=oci -t quay.io/<user_name>/<repository_name>:<tag_name> .
+podman build --format=oci --squash -t quay.io/<user_name>/<repository_name>:<tag_name> .
 podman push quay.io/<user_name>/<repository_name>:<tag_name>
 ```
 
-> [!TIP]
-> When uploading your container image, if your repository is private, ensure you
```

Comment:
why not keep?

Reply:
I am minifying the MobileNet example, assuming the reader would read from top to bottom and would already know this from the Granite example.

Reply:
who knows? :-)